client/components/marketing-survey/cancel-purchase-form/index.jsx

```diff
@@ -1250,12 +1008,20 @@ export default connect(
 		isChatActive: hasActiveHappychatSession( state ),
 		isAtomicSite: isSiteAutomatedTransfer( state, purchase.siteId ),
 		isImport: !! getSiteImportEngine( state, purchase.siteId ),
-		isJetpack: isJetpackPlan( purchase ) || isJetpackProduct( purchase ),
 		downgradePlanPrice: getDowngradePlanRawPrice( state, purchase ),
 		supportVariation: getSupportVariation( state ),
 		site: getSite( state, purchase.siteId ),
 		willAtomicSiteRevert: willAtomicSiteRevertAfterPurchaseDeactivation( state, purchase.id ),
 		atomicTransfer: getAtomicTransfer( state, purchase.siteId ),
+		canInstallPlugins: siteHasFeature( state, purchase.siteId, WPCOM_FEATURES_INSTALL_PLUGINS ),
+		canInstallPremiumPlugins: siteHasFeature(
```

Comment:
Maybe this could be `canInstallPurchasedPlugins`?

Reply:
Not sure if it's worth separating `can*` nomenclature for user capabilities from `has*` nomenclature for features? I tried to stick with `has*` in the past to avoid confusion.